src/engine/enginebuffer.cpp

```diff
@@ -1136,7 +1151,7 @@ void EngineBuffer::processSlip(int iBufferSize) {
     bool enabled = static_cast<bool>(load_atomic(m_slipEnabled));
     if (enabled != m_bSlipEnabledProcessing) {
         m_bSlipEnabledProcessing = enabled;
-        if (enabled) {
+        if (enabled || m_slipCancelled.fetchAndStoreAcquire(0)) {
```

Comment:
Please lets make it rock solid.
A script might set just 1, without reset it to 0.  
A event driven design will work just set 1 if it is already at 1.